features/tracker-allowlist.json

```
@@ -308,6 +308,20 @@
                     }
                 ]
             },
+            "anyclip.com": {
+                "rules": [
+                    {
+                        "rule": "player.anyclip.com/anyclip-widget/lre-widget/.*/src/lre.js",
```

```suggestion
                        "rule": "player.anyclip.com/anyclip-widget/lre-widget",
```

Thanks, @muodov -- unfortunately while the blocklist uses wildcards, afaik the config can't so the rule here should probably just be `player.anyclip.com/anyclip-widget/lre-widget`

---

@muodov good question! Could be only some platforms, could be I'm remembering incorrectly (this seems to be the piece of info I'm always trying to look up). @SlayterDev can you help here?

---

If we have other examples of this I'd say its generally fine. You might test it on a couple other platforms to be sure. I think @jdorweiler or @GuiltyDolphin will have more context around the tracker allowlist.